navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/NavigationService.java

```diff
@@ -36,6 +33,7 @@
  * @since 0.1.0
  */
 @Experimental
+@SuppressWarnings( {"MissingPermission"})
```

Comment:
Instead of attaching this to the whole class, let's put on the line(s) where it's actually need to avoid masking other instances.